### DISCUSSION TITLE
Support for signal inputs in Angular

### DISCUSSION THREAD

**Discussion by iabu94** [storybookjs/storybook#26859] [Ideas]
### Is your feature request related to a problem? Please describe.

Since I have updated my current project to angular v17 and using the signal based inputs, the storybook not supported for it. throwing errors when building.

### Describe the solution you'd like

provide the support for signal inputs asap

### Are you able to assist to bring the feature to reality?

yes, I can

**Comment by Marklb** [comment]
Could you provide a reproduction or what the error is, if you can't provide a repro? If it is a simple scenario, a Stackblitz can be quickly made from https://storybook.new

I made a very simple Stackbltz example, to see if I get an error and it seems to work fine: https://stackblitz.com/edit/github-mbwd5e

For reference, my Stackblitz example is just the following:
```ts
import { CommonModule } from '@angular/common';
import { Component, input } from '@angular/core';

@Component({
  selector: 'signal-ex',
  standalone: true,
  imports: [CommonModule],
  template: `Input: {{ thing() }}`,
  styleUrls: [],
})
export class SignalExComponent {
  thing = input('initial value');
}
```

```ts
import type { Meta, StoryObj } from '@storybook/angular';

import { SignalExComponent } from './signal-ex.component';

const meta: Meta<SignalExComponent> = {
  component: SignalExComponent,
  tags: ['autodocs'],
};

export default meta;
type Story = StoryObj<SignalExComponent>;

export const Primary: Story = {
  args: {
    thing: 'updated value',
  },
};
```

**Reply by jyannessa-mtx** [reply]
If you downgrade from that alpha version to 8.0.10 you will get [the error](https://stackblitz.com/edit/github-mbwd5e-jxkknb?file=src%2Fapp%2Fsignal-ex%2Fsignal-ex.stories.ts), so supposedly the fix is to upgrade to >=8.1.0
